src/modules/function/declaration_utils.rs

```diff
@@ -6,7 +6,11 @@ use crate::utils::cc_flags::{CCFlags, get_ccflag_name};
 use crate::utils::context::Context;
 use crate::utils::function_interface::FunctionInterface;
 
-pub fn skip_function_body(meta: &mut ParserMetadata, declared_failable: bool, returns_tok: &Option<Token>) -> Result<(usize, usize, bool), Failure> {
+pub fn skip_function_body(
+    meta: &mut ParserMetadata,
+    declared_failable: bool,
+    returns_tok: &Option<Token>
```

Comment:
I generally add a trailing comma to the last of a set of function arguments (also `match` arms and other comma separated items) on multiple lines.  This is for two reasons: (i) it makes subsequent pull request diffs nicer to review, and (ii) it makes it simpler to run keystroke macros.